File: solutions/8kyu/quarter_of_the_year.js
// Найти в какой четверти месяц, по его номеру
// В каждой четверти 3 месяца, делим на 3 - получаем четверть + округляем.

const quarterOf = (month) => {
  return Math.ceil(month / 3)
};

// BEST practice
// const quarterOf = m => Math.ceil(m/3);

console.log(quarterOf(11)) //=> 4

File: solutions/8kyu/quarter_of_the_year.rb
# Найти в какой четверти месяц, по его номеру

def quarter_of(month)
  if month > 6
    month > 9 ? 4 : 3
  else
    month > 3 ? 2 : 1
  end
end


# BEST practice
# В каждой четверти 3 месяца, делим на 3 - получаем четверть + округляем...
# def quarter_of(month)
#    (month/3.0).ceil
# end

p quarter_of(11) #=> 4
